src/app/client/src/app/app.component.spec.ts

```diff
@@ -223,4 +223,10 @@ const maockOrgDetails = { result: { response: { content: [{hashTagId: '1235654',
     component.ngOnInit();
     expect(component.showFrameWorkPopUp).toBeTruthy();
   });
+
+  it('Should return proper object by calling make UTM session', () => {
```

Comment:
this testcase is not correct. This should be triggered from activated route. Note by calling component.makeUTMSession

Reply:
Done modification for the same in test cases.